README.adoc

```diff
@@ -598,8 +617,9 @@ deploy.yaml
 include::finish/deploy.yaml[]
 ----
 
-Now that the `system` microservice is running, you will package and deploy the `inventory` microservice, which adds the properties from the `system` 
-microservice to the `inventory`. This process demonstrates how communication can be established between pods inside a cluster.
+Now that the `system` microservice is running, you will package and deploy the `inventory` microservice,
+which adds the properties from the `system` microservice to the `inventory`.
+This process demonstrates how communication can be established between pods inside a cluster.
```

Comment:
This process demonstrates how communication can be established between pods inside a cluster.
--->
This process demonstrates how to establish communication between pods inside a cluster.